modules/deploy/pages/deployment-option/cloud/aws-privatelink.adoc

```diff
@@ -0,0 +1,403 @@
+
+[,bash]
+----
+aws ec2 create-security-group --region $REGION --profile $PROFILE --vpc-id $CLIENT_VPC_ID \
+    --description "Redpanda endpoint service client security group" \
+    --group-name "${CLUSTER_ID}-sg"
+SECURITY_GROUP_ID=<security_group_id>
+----
+
+=== Add security group rules
+
+This setting is based on the assumption that the Redpanda broker count is three. If you are not using three brokers, modify the example below:
+
+* Replace `32094` with `32092 + <broker_count-1>`.
+* Replace `35084` with `35082 + <broker_count-1>`.
+
+[,bash]
+----
+aws ec2 authorize-security-group-ingress --region $REGION --profile $PROFILE \
+    --group-id $SECURITY_GROUP_ID \
+    --protocol "tcp" \
+    --port 30292 \
+    --cidr 0.0.0.0/0
+aws ec2 authorize-security-group-ingress --region $REGION --profile $PROFILE \
+    --group-id $SECURITY_GROUP_ID --protocol "tcp" \
+    --port 30081 \
+    --cidr 0.0.0.0/0
+aws ec2 authorize-security-group-ingress --region $REGION --profile $PROFILE \
+    --group-id $SECURITY_GROUP_ID --protocol "tcp" \
+    --port 30282 \
+    --cidr 0.0.0.0/0
+# Adjust the port 32094 if the Redpanda broker count is not 3.
+aws ec2 authorize-security-group-ingress --region $REGION --profile $PROFILE \
+    --group-id $SECURITY_GROUP_ID \
+    --protocol "tcp" \
+    --port 32092-32094 \
+    --cidr 0.0.0.0/0
+# Adjust the port 35084 if the Redpanda broker count is not 3.
+aws ec2 authorize-security-group-ingress --region $REGION --profile $PROFILE \
+    --group-id $SECURITY_GROUP_ID \
+    --protocol "tcp" \
+    --port 35082-35084 \
+    --cidr 0.0.0.0/0
+----
+
+=== Create VPC subnet
+
+Run the following command, specifying the subnet availability zone (for example, `usw2-az1`). You need the subnet ID `subnet_id` from the command output to <<create-vpc-endpoint,create a VPC endpoint>>.
+
+[,bash]
+----
+aws ec2 create-subnet --region $REGION --profile $PROFILE --vpc-id $CLIENT_VPC_ID \
+    --availability-zone <zone> \
+    --cidr-block 10.0.1.0/24
+SUBNET_ID=<subnet_id>
+----
+
+=== Create VPC endpoint
+
+[,bash]
+----
+aws ec2 create-vpc-endpoint \
+    --region $REGION --profile $PROFILE \
+    --vpc-id $CLIENT_VPC_ID \
+    --vpc-endpoint-type "Interface" \
+    --ip-address-type "ipv4" \
+    --service-name $PL_SERVICE_NAME \
+    --subnet-ids $SUBNET_ID \
+    --security-group-ids $SECURITY_GROUP_ID \
+    --private-dns-enabled
+----
+
+== Access Redpanda services through VPC endpoint
+
+You can access Redpanda services such as xref:manage:schema-reg/schema-reg-overview.adoc[Schema Registry] and xref:develop:http-proxy.adoc[HTTP Proxy] from the client VPC, for example, from an EC2 instance in the VPC.
+
+=== Access Kafka API seed service
+
+Use port `30292` to access the Kafka API seed service.
+
+[,bash]
+----
+export REDPANDA_BROKERS=rp-endpoint.$CLUSTER_DOMAIN:30292
```

Comment:
@paulzhang97 Can we specify what `rp-endpoint` is? Where can a user find this value?

Reply:
It is private link domain prefix. It is hardcoded as `rp-endpoint`.